include/blast/math/views/Submatrix.hpp

```diff
@@ -6,3 +6,283 @@
 
 #include <blast/math/views/submatrix/BaseTemplate.hpp>
 #include <blast/math/views/submatrix/Panel.hpp>
+#include <blast/math/TypeTraits.hpp>
+
+#include <type_traits>
+
+
+namespace blast
+{
+    /**
+     * @brief Submatrix view of a matrix
+     *
+     * @tparam MT viewed matrix type
+     * @tparam AF whether the submatrix lop left element is aligned
+     */
+    template <Matrix MT, AlignmentFlag AF>
+    class Submatrix
```

Comment:
I like the name :+1:

Reply:
Can you explain in the comments why `mdspan` wasn't used as an implementation?

Reply:
The main reason is that `std::mdspan` does not know whether it is aligned or not.